frontends/bmd/src/pages/review_page.tsx

```diff
@@ -546,11 +548,29 @@ export function ReviewPage(): JSX.Element {
         <Sidebar
           footer={
             <React.Fragment>
-              <SettingsTextSize
-                userSettings={userSettings}
-                setUserSettings={setUserSettings}
-                sidebarWrapper
-              />
+              <ButtonFooterLandscape>
+                {/* <SegmentedButton>
+                  <Button
+                    primary
+                    onPress={() => {}}
+                    aria-label="Change Language"
+                  >
+                    Español
+                  </Button>
+                  <Button onPress={() => {}} aria-label="Change Language">
+                    English
+                  </Button>
+                </SegmentedButton> */}
+                {/* <Button onPress={() => {}} aria-label="Change Language">
+                  English
+                </Button> */}
```

Comment:
Are these commented out buttons meant to be in a different order for landscape and portrait mode respectively?

Reply:
Cleaned up by creating a component.